### ISSUE TITLE
Wrong version format in requirements for Data-HexDump-Range

### ISSUE THREAD

**Issue by zprogd** [TrackingSoft/Kafka#36]
Data-HexDump-Range has letter 'v' in its versions: v0.13.72, v0.13.71, v0.13.59, v0.12.57

In META.json and META.yml version without 'v':
"Data::HexDump::Range" : "0.13"

**Comment by asolovey** [comment]
We do not care about specific version of Data::HexDump::Range, so I'll remove explicit version requirement from our Makefile.PL in the next release.

However, the confusion between 0.13 and v0.13.72 may be caused by Data::HexDump::Range, here is an excerpt from its [META.json](http://cpansearch.perl.org/src/NKH/Data-HexDump-Range-0.13.72/META.json):
```json
   "provides" : {
      "Data::HexDump::Range" : {
         "file" : "lib/Data/HexDump/Range.pm",
         "version" : "0.13"
      }
   },
   .....
   "version" : "v0.13.72"
```
So the module that we require has version 0.13, but its distribution has a different version.

**Comment by asolovey** [comment]
Fixed in be02b67d713eee19a1fd8ec9b719c16bb9e65f5b
